community_supplied/Kernel/kernel-tcpip/check-arp-cache-iri-count-iagent.rule

```diff
@@ -0,0 +1,181 @@
+                argument percent_val {
+                    mandatory;
+                }
+            }
+            sensor arp-iri-cache-count-sensor {
+                synopsis "Sensor to check arp iri cache count";
+                description "Sensor to check arp iri cache count";
+                iAgent {
+                    file arp_iri_cache.yml;
+                    table checkArpIriCacheTable;
+                    frequency 1m;
+                }
+            }
+            field arp-cache-iri-critical-percent {
+                constant {
+                    value "{{arp-cache-iri-critical-percent-input}}";
+                }
+                description "arp iri cache percentage critical limit";
+            }
+            field arp-cache-iri-warning-percent {
+                constant {
+                    value "{{arp-cache-iri-warning-percent-input}}";
+                }
+                description "arp iri cache percentage warning limit";
+            }
+            field nbrcache-iri-cnt {
+                sensor arp-iri-cache-count-sensor {
+                    path arpcache_iri_cnt;
+                }
+                type integer;
+                description "Current arp iri cache count in the system";
+            }
+            field nbrcache-iri-drop-cnt {
+                sensor arp-iri-cache-count-sensor {
+                    path arpcache_iri_drop_cnt;
+                }
+                type integer;
+                description "Maximum arp iri cache drop count in the system";
+            }
+            field nbrcache-iri-max {
+                sensor arp-iri-cache-count-sensor {
+                    path arpcache_iri_max_cnt;
+                }
+                type integer;
+                description "Maximum arp iri cache count in the system";
+            }
+            field output-critical {
+                formula {
+                    user-defined-function {
+                        function-name arp-cache-iri-status;
+                        argument max_val "$nbrcache-iri-max";
+                        argument actual_val "$nbrcache-iri-cnt";
+                        argument percent_val "$arp-cache-iri-critical-percent";
+                    }
+                }
+                type integer;
+            }
+            field output-warning {
+                formula {
+                    user-defined-function {
+                        function-name arp-cache-iri-status;
+                        argument max_val "$nbrcache-iri-max";
+                        argument actual_val "$nbrcache-iri-cnt";
+                        argument percent_val "$arp-cache-iri-warning-percent";
+                    }
+                }
+                type integer;
+            }
+            trigger report-arp-iri-cache-count-iagent {
+                synopsis "Trigger to check arp iri cache count";
+                description "Trigger to check arp iri cache count";
+                frequency 1m;
```

Comment:
Always ensure trigger frequency is atleast 1.5 times of sensor frequency. Modify trigger frequency

Reply:
Addressed